aries-site/src/data/wcag/components.json

```diff
@@ -862,6 +862,518 @@
+        "label": "Timing Adjustable",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.1",
+        "label": "Bypass Blocks",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.2",
+        "label": "Page Titled",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.3",
+        "label": "Focus Order",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.4",
+        "label": "Link Purpose (In Context)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.5",
+        "label": "Multiple Ways",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.6",
+        "label": "Headings and Labels",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.7",
+        "label": "Focus Visible",
+        "status": "failed",
+        "notes": "TextInput with suggestions, the treatment of the suggestion that is in focus is not meeting requirement."
+      },
+      {
+        "rule": "2.4.8",
+        "label": "Location",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.9",
+        "label": "Link Purpose (Link Only)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.10",
+        "label": "Section Headings",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.11",
+        "label": "Focus Not Obscured (Minimum)",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.4.12",
+        "label": "Focus Not Obscured (Enhanced)",
+        "status": "failed",
```

Comment:
I think this is passing, looking back at the textInput PR I realized the comment I made here was confusing. I think 2.4.7 is failing but 2.4.11 and 2.4.12 are passing.

Reply:
Thanks for clarifying, that makes more sense. Updated.